packages/api-client-core/src/ClientOptions.ts

```diff
@@ -10,6 +10,7 @@ export interface ClientOptions {
 
 /** Options to configure a specific browser-based authentication mode */
 export interface BrowserSessionAuthenticationModeOptions {
+  initialToken?: string;
```

Comment:
Can you add a doc comment to explain what this does / if anyone else should use it?